backend/src/main/java/com/votogether/domain/post/entity/Post.java

```diff
@@ -124,18 +125,31 @@ public Vote makeVote(Member member, PostOption postOption) {
                 .build();
     }
 
-    private void validateDeadLine() {
+    public void validateDeadLine() {
         if (isClosed()) {
-            throw new IllegalStateException("게시글이 이미 마감되었습니다.");
+            throw new BadRequestException(PostExceptionType.POST_CLOSED);
         }
     }
 
     private void validatePostOption(PostOption postOption) {
         if (!hasPostOption(postOption)) {
-            throw new IllegalArgumentException("해당 게시글에서 존재하지 않는 선택지 입니다.");
+            throw new BadRequestException(PostExceptionType.POST_OPTION_NOT_FOUND);
         }
     }
 
+    public void validateHalfDeadLine() {
+        final Duration betweenDuration = Duration.between(getCreatedAt(), this.deadline);
+        final LocalDateTime midpoint = getCreatedAt().plus(betweenDuration.dividedBy(2));
```

Comment:
`P3`
```suggestion
        final LocalDateTime earlyClosingTime = getCreatedAt().plus(betweenDuration.dividedBy(2));
```
변수명을 조금 더 명확하게 해도 좋을 것 같아요 :)